include/SPTPersistentCache/SPTPersistentCache.h

```diff
@@ -37,6 +37,8 @@ FOUNDATION_EXPORT double SPTDataLoaderVersionNumber;
 FOUNDATION_EXPORT const unsigned char SPTDataLoaderVersionString[];
 #endif // SPT_BUILDING_FRAMEWORK
 
+uint64_t spt_uint64rint(double value);
```

Comment:
This shouldn’t be exposed in our public API IMO.

Instead I’d prefer if we could move this to a private header file in `Sources/`. So instead create `Sources/SPTPersistentCacheTypeUtilities.h` and have it contain:

```objc
/**
 * Converts the given `double` _value_ to an `uint64_t` value.
 *
 * @param value The value as a `double`.
 * @return The value as an `uint64_t`.
 */
NS_INLINE uint64_t spt_uint64rint(double value)
{
    return (uint64_t)llrint(value);
}
```

Then remove this declaration and the definition in `Sources/SPTPersistentCache.m`.